example/lib/main.dart

```diff
@@ -248,7 +238,7 @@ class _DialScreenState extends State<DialScreen> with WidgetsBindingObserver {
   }
 
   void pushToCallScreen() {
-    Navigator.of(context, rootNavigator: true).push(MaterialPageRoute(
-        fullscreenDialog: true, builder: (context) => CallScreen()));
+    Navigator.of(context, rootNavigator: true)
+        .push(MaterialPageRoute(fullscreenDialog: true, builder: (context) => CallScreen()));
```

Comment:
Let's reduce unnecessary changes, that which default linter doesn't suggest/require.